dagmc_stats/DagmcStats.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, filename, populate=False):
         self._vol_data = pd.DataFrame(
             columns=['vol_eh', 'surf_per_vol', 'coarseness'])
         self._vol_data = self._vol_data.set_index('vol_eh')
+        
+        self.native_ranges = self.__set_native_ranges(entity)
```

Comment:
Do we need to pass in the entity types, or just use the ones defined as a class variable?

Reply:
don't need to pass `entity` here if we are using `self.entity_types` as a class variable.